libraries/mysql_service_manager_systemd.rb

```diff
@@ -4,7 +4,7 @@ class MysqlServiceManagerSystemd < MysqlServiceBase
 
     provides :mysql_service_manager, platform: 'fedora'
 
-    provides :mysql_service_manager, platform: %w(redhat centos scientific) do |node| # ~FC005
+    provides :mysql_service_manager, platform: %w(redhat centos scientific oracle) do |node| # ~FC005
       node['platform_version'].to_f >= 7.0
```

Comment:
all kinds of wrong here, it should ideally use platform_family of `rhel` or `fedora` and then check if systemd is installed but the whole thing should boil down to:

```ruby
  provides :service, os: "linux" do |node|
    Chef::Platform::ServiceHelpers.service_resource_providers.include?(:systemd)
  end
```

I should really add `systemd?` to core from chef-sugar and replace that ugliness.